installation/resources/installer-cr.yaml.tpl

```diff
@@ -24,16 +24,16 @@ spec:
       namespace: "kyma-installer"
     - name: "istio-kyma-patch"
       namespace: "istio-system"
-    - name: "knative-serving-init"
-      namespace: "knative-serving"
-    - name: "knative-serving"
-      namespace: "knative-serving"
+    # - name: "knative-serving-init"
```

Comment:
why changes in this files?

Reply:
That was a change for local testing, should not have been commited :D